docs/evaluation/simulate.rst

```diff
@@ -186,15 +282,19 @@ numbers as background value and then performing a Poisson draw on them
 again gives a higher level of noise than a real observation would
 have. To avoid this problem, a background model may be used instead::
 
-  >>> from sherpa.models.basic import Const1D
-  >>> bkgmdl = Const1D('bmdl')
-  >>> bkgmdl.c0 = 2
-  >>> fake_pha(data, mdl, add_bkgs=True, bkg_models={'1': bkgmdl})
+   >>> from sherpa.models.basic import Const1D
+   >>> bkgmdl = Const1D('bmdl')
+   >>> bkgmdl.c0 = 2
+   >>> fake_pha(data, mdl, add_bkgs=True, bkg_models={'1': bkgmdl})  # doctest: +SKIP
 
 The keys in the `bkg_models` dictionary are the identifiers of the
 backgrounds. Above, we loaded a background with the default identifier
 (which is ``1``).
 
+.. todo::
+   Looks like we need `instmdl` here, but does this mean that
+   bkg_models needs to be sent `resp(bkgmdl)` ?
+
 
 More than one background
 ^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
I can't leave a comment on the exact line below becaue it's not been changed, but if there are multiple background components, theu will now be added, not averaged:
```Python
        for bkg_id in data.background_ids:
            cts = data.get_background(bkg_id).counts
            scale = data.get_background_scale(bkg_id, units="counts")
            model_prediction += scale * cts
```